### PR TITLE
New master merge

### LINKED ISSUE #738: Gentoo VMs do not have enough disk space.
When you make the disk image for it, it is not enough to install Gentoo. I don't know how much space is needed for Gentoo(I tried in installing it last night for the first time) but I keep getting the no more disk space error. Is it meant only to be booted live or is their a real/temp fix for this?

### PR DESCRIPTION
**Pull request by zen0bit**
# Should have almost all other PRs included with exceptions:
### Not yet ready PRs and my test PRs with linting, actions etc.
#848 - No matter what I tried, don't work for me
#947 - Instead rised VM sizes for kali and gentoo only..

# Merging this should close:
Close #738 #944 #971 #973 #974 #982 Issues
Close #748 #847 #870 #886 #887 #890 #901 #902 #906 #911 #913 #923 #932 #940 #948 #949 #950 #972 #981 included PRs
Close #895 #952 - Documentation - Rebuild after PLS

PS: Am I missed something?
PPS: Will look more into issues...

### CODE REVIEW

**Comment by TuxVinyards** [comment]
Good work 👍

**Comment by zen0bit** [comment]
I am not done yet... :rofl:

File: quickget
Comment (lj3954) [inline quickget:1]: This distro appears to have different latest releases for KDE and GNOME. The release you put is the latest release of GNOME CachyOS, but is invalid for KDE. I think instead, you should just put 'latest' and handle everything else in the get_cachyos function.

Comment (lj3954) [reply quickget:1]: Here's the fixed 'get_cachyos' function to go along with that.

```bash
function get_cachyos() {
    local HASH=""
    local latest_release
    latest_release=$(curl "https://mirror.cachyos.org/ISO/${EDITION}/" | grep -Po '(?<=">)[0-9]+(?=/</a>)' | tail -n 1)

    local ISO="cachyos-${EDITION}-linux-${latest_release}.iso"
    local URL="https://mirror.cachyos.org/ISO/${EDITION}/${latest_release}"
    HASH=$(wget -q -O- "${URL}/${ISO}.sha256" | cut -d' ' -f1)
    echo "${URL}/${ISO} ${HASH}"
}
```

File: quickget [hunk 1]
```diff
@@ -543,7 +543,7 @@ function editions_bodhi() {
 }
 
 function releases_cachyos() {
-    echo 230813
+    echo 240121
```
Comment (lj3954) [inline quickget:546]:
```suggestion
    echo latest
```

**Review by lj3954 — CHANGES_REQUESTED** [review]

**Pull request merged**